draft-ietf-dnsop-svcb-https.md

```diff
@@ -763,6 +759,33 @@ in the initial response.  As a performance optimization, if some of the SVCB
 records in the response can be used without requiring additional DNS queries,
 the client MAY prefer those records, regardless of their priorities.
 
+# SVCB-compatible 
+
+An RR type is called "SVCB-compatible" if it permits an implementation that is
+identical to SVCB in its:
+
+* RDATA presentation format
+* RDATA wire format
+* IANA registry used for SvcParamKeys
+* Authoritative server Additional Section processing
+* Recursive resolution process
+* Relevant Class (i.e. Internet ("IN") {{!RFC1035}})
+
+This allows authoritative and recursive DNS servers to apply identical
+processing to all SVCB-compatible RR types.
+
```

Comment:
If we have a SVCB-compatible record like "NS2" that itself is used for recursive to authoritative logic, it's possible that this might not be identical in-terms of Recursive resolution process (or Auth server processing).  How do we allow for such in the future ?  Should those be "unless specified otherwise"?

Reply:
I think a SVCB-based type that requires some additional logic is "partially SVCB-compatible".  That may be a good choice in the future, but I don't think we need to define it here.